chart_review/__init__.py

```diff
@@ -1,3 +1,3 @@
 """Chart Review public entry point"""
 
-__version__ = "2.1.0"
+__version__ = "2.1.1"
```

Comment:
Does not matter at all really, but I'd say a new flag and change of default behavior falls squarely under the "feature" banner and could safely result in a minor bump instead of a patch bump. But :shrug: